components/sections/Navigation/index.tsx

```diff
@@ -33,11 +33,10 @@ const Navigation = () => {
 			>
 				<div className='flex flex-col gap-4 xl:flex-row xl:gap-8'>
 					<Link
-						className='nav-link-shadow transition-all text-sky-300'
+						className='nav-link-shadow text-sky-300 transition-all'
 						href='/cch'
 						onClick={() => {
 							trackEvent('homepage_mainnav_cch')
-							setOpen(false)
```

Comment:
I'm not sure why clicking on a link in the pop-up menu should have closed it. I might be missing something here.

Reply:
Wouldn't you want the menu to close and let you read the page you clicked on?
The only time you'd want it to stay open is when Ctrl-clicking as in https://github.com/shuttle-hq/www/issues/196#issuecomment-1837116890 right? If so, this onClick handler might be able to look for [ctrlKey](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/ctrlKey) and close the menu based on it's value.

Reply:
Ah, I see.